drivers/start_date_dq/app/models/start_date_dq/report.rb

```diff
@@ -70,11 +95,40 @@ def report_scope
       scope = filter_for_range(scope)
       scope = filter_for_project_type(scope, all_project_types: false)
       scope = filter_for_projects(scope)
+      scope = filter_for_cocs(scope)
       scope
     end
 
     def report_scope_source
       GrdaWarehouse::ServiceHistoryEnrollment.entry.joins(:enrollment)
     end
+
+    def day_ranges
+      {
+        '<0 days': (-Float::INFINITY..-1),
+        '0-30 days': (-1..30),
+        '31-90 days': (30..90),
+        '91-180 days': (90..180),
+        '181+ days': (180..Float::INFINITY),
```

Comment:
Any reason not to use `gteq` for the lower bound? I'm worried about the lower bound in the text and the lower bound in the range not matching.

Reply:
I set it up this way so I could use `range.overlaps?(other_range)` when merging the ranges in the `collapse_ranges` function. I agree with you though, and I can easily add a helper function to check if ranges are adjacent instead.

Now that I'm looking at this again, I wonder if I was over-optimizing and should get rid of the `collapse_ranges` function altogether. It simplifies the SQL so it's not computing the datediff so many times, but it might be overkill and not worth the added code complexity. Let me know if you have an opinion on that.